### DISCUSSION TITLE
System Tray Depth

### DISCUSSION THREAD

**Discussion by simonbramble** [valinet/ExplorerPatcher#3428] [Q&A]
My Taskbar/System Tray is double height. It suddenly went like that yesterday. I have tried hiding icons in my System Tray, but this does not seem to help and I cannot find out how to make it normal height. I can only conclude it is because my Clock has the day in it, which is making it higher than normal. How do I either remove "Thursday" from the clock (see image below) so it does not occupy 3 rows, or make the Taskbar/System tray lower in height?

![Untitled](https://github.com/valinet/ExplorerPatcher/assets/46623385/f149c163-8d37-4192-870a-e6006335adb3)

**Comment by pyrates999** [comment]
Right click on the taskbar and untick **lock the taskbar**.  Then you can resize the taskbar.  Then right click on the taskbar and untick **lock the taskbar**.

**Comment by simonbramble** [comment]
Perfect. Thanks. I tried everything... except the obvious.

Great program by the way - Thanks for writing it

Simon

**Comment by pyrates999** [comment]
The author of EP appreciates your comments.
